packages/config/src/projects/layer2s/scroll.ts

```diff
@@ -173,6 +173,11 @@ export const scroll: Layer2 = {
         tokens: ['USDC'],
         ...upgradesScrollMultisig,
       }),
+      discovery.getEscrowDetails({
```

Comment:
sorry just found during testing, that adding more gateways might do the same steps as this PR: https://github.com/l2beat/l2beat/pull/3754